packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -1,70 +1,109 @@
-import React, { ReactElement, useContext, useEffect } from 'react';
+import React, { ReactElement, useState } from 'react';
 import classNames from 'classnames';
-import FilterMenu from './FilterMenu';
-import CloseIcon from '../icons/Close';
-import PlusIcon from '../icons/Plus';
-import { menuItemClassNames } from '../multiLevelMenu/MultiLevelMenuMaster';
-import useFeedSettings from '../../hooks/useFeedSettings';
-import AuthContext from '../../contexts/AuthContext';
-import AlertContext from '../../contexts/AlertContext';
-import { useMyFeed } from '../../hooks/useMyFeed';
-import CreateFeedFilterButton from '../CreateFeedFilterButton';
+import { ModalProps, StyledModal } from '../modals/StyledModal';
+import SidebarList from '../sidebar/SidebarList';
+import HashtagIcon from '../icons/Hashtag';
+import FilterIcon from '../icons/Filter';
+import BlockIcon from '../icons/Block';
+import TagsFilter from './TagsFilter';
+import { TagCategoryLayout } from './TagCategoryDropdown';
+import AdvancedSettingsFilter from './AdvancedSettings';
+import BlockedFilter from './BlockedFilter';
+import CloseButton from '../modals/CloseButton';
+import styles from './FeedFilters.module.css';
+import ArrowIcon from '../icons/Arrow';
+import { Button } from '../buttons/Button';
 
-interface FeedFiltersProps {
-  directlyOpenedTab?: string;
-  isOpen?: boolean;
-  onBack?: () => void;
-}
+const items = [
+  {
+    title: 'Manage tags',
+    icon: <HashtagIcon />,
+    component: <TagsFilter tagCategoryLayout={TagCategoryLayout.Settings} />,
+  },
+  {
+    title: 'Advanced',
+    icon: <FilterIcon />,
+    component: <AdvancedSettingsFilter />,
+  },
+  { title: 'Blocked items', icon: <BlockIcon />, component: <BlockedFilter /> },
+];
+
+const HEADER_HEIGHT = 64;
 
 export default function FeedFilters({
-  directlyOpenedTab,
   isOpen,
-  onBack,
-}: FeedFiltersProps): ReactElement {
-  const { alerts, updateAlerts } = useContext(AlertContext);
-  const { user } = useContext(AuthContext);
-  const { hasAnyFilter } = useFeedSettings();
-  const { shouldShowMyFeed } = useMyFeed();
+  onRequestClose,
+  ...props
+}: ModalProps): ReactElement {
+  const [contentRef, setContentRef] = useState<HTMLDivElement>();
+  const [isNavOpen, setIsNavOpen] = useState(false);
+  const [display, setDisplay] = useState<string>(items[0].title);
 
-  useEffect(() => {
-    if (isOpen && alerts?.filter && hasAnyFilter && user && !shouldShowMyFeed) {
-      updateAlerts({ filter: false });
+  const onNavClick = (active: string) => {
+    setDisplay(active);
+    if (isNavOpen) {
+      setIsNavOpen(false);
     }
-  }, [isOpen, alerts, user, hasAnyFilter]);
+  };
+
+  const getHeight = () => {
```

Comment:
Just a question more general:
Why do we need to calculate the height dynamic. I think in our new modals everything is either 100% or a fixed height right?

Reply:
This is so the scrollbar is not hitting the header. With that mentioned, let me check how it would look like when the `position` is `sticky` or `fixed` since in the design, it should be always on top when scrolling.

Reply:
Sticky worked beautifully! Thank you for questioning this! 🚢